vtm/src/org/oscim/layers/tile/TileManager.java

```diff
@@ -729,4 +748,21 @@ public void setZoomLevel(int zoomLevelMin, int zoomLevelMax) {
         mMinZoom = zoomLevelMin;
         mMaxZoom = zoomLevelMax;
     }
+
+    /**
+     * Allows loading tiles at specified zoom level if higher zoom levels are requested.
+     */
+    public void addZoomLimit(int zoomLimit) {
+        if (zoomLimit > mMinZoom && zoomLimit < mMaxZoom) {
+            mZoomLimits.add(zoomLimit);
+            Collections.sort(mZoomLimits);
+        }
+    }
+
+    /**
+     * Remove zoom limit. Tiles of that zoom level won't be requested anymore on higher zoom levels.
+     */
+    public void removeZoomLimit(int zoomLimit) {
+        mZoomLimits.remove(zoomLimit);
```

Comment:
That equals to `List.remove(int index)` and will remove the element at the specified position, instead of the actual element.

Reply:
Probably can use:
```java
mZoomLimits.remove(Integer.valueOf(zoomLimit));
```

Reply:
Yes thats right! Thx. Can you push it?